x-pack/platform/plugins/shared/dataset_quality/public/components/failure_store/failure_store_warning.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLink, EuiCallOut } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { useKibanaContextForPlugin } from '../../utils';
+
+export const FailureStoreWarning = () => {
+  const { services } = useKibanaContextForPlugin();
+
+  const noAccessToFailureStoreWarningDescription = (
+    <FormattedMessage
+      id="xpack.datasetQuality.noAccessToFailureStore.description"
+      defaultMessage="{description}"
+      values={{
+        description: (
+          <FormattedMessage
+            id="xpack.datasetQuality.noAccessToFailureStore.warning"
+            defaultMessage="Documents that fail at ingestion are sent to the failure store. You don't have the required privileges to access failure store. Contact your administrator. {learnMoreLink}"
```

Comment:
"fail at ingestion" is not ideal, but changing the verb will make it harder to map to the "Failed docs" column so it probably needs to stay. here's an option to make it a little clearer, and a small tweak because there is a failure store per index / data stream

```suggestion
            defaultMessage="Documents that fail to be ingested are sent to the failure store. You don't have the required privileges to access failure stores. Contact your administrator. {learnMoreLink}"
```